### PR TITLE
Feat/job board

### PR DESCRIPTION
**Pull request by theetrain**
Add minimally-viable job board. See #68 

## Sample

screen recording 2017-10-11 at 10 49 pm

## Desktop

image

## Mobile

image

### CODE REVIEW

File: assets/js/jobs.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      var job = jobs.find(findJob)
+      var jobIndex = jobs.findIndex(findJob)
+      if (!!job === false) {
+        return
+      }
+
+      this.expand(
+        jobIndex,
+        document.title,
+        window.location.pathname + window.location.search
```
Comment (theetrain) [inline assets/js/jobs.js]: This parameter isn't used 😱

File: assets/js/jobs.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  }
+
+  // "Expanded job description" wrapper
+  var jobDescription = (function (el, title, body) {
+    var element = el
+    var elTitle = title
+    var elBody = body
+
+    return {
+      setTitle: function (title) {
```
Comment (theetrain) [inline assets/js/jobs.js]: This can be combined with `setBody`

File: assets/js/jobs.js [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    return {
+      set: function (value) {
+        element.innerHTML = value
+        return element
+      }
+    }
+  })(document.getElementById('job-body'))
+
+  // Single table row containing job details
+  var jobItem = function (element, index) {
```
Comment (theetrain) [inline assets/js/jobs.js]: `jobItem` can be `jobRowHtml`
`element` property can be `job`

File: assets/js/jobs.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+        document.title,
+        window.location.pathname + window.location.search
+      )
+      jobDescription.close()
+      jobTable.open()
+    },
+    listJobs: function () {
+      var jobCollection = ''
+
+      jobs.forEach(function (element, index) {
```
Comment (theetrain) [inline assets/js/jobs.js]: Can be simplified:
```js
var jobTableRowsHTML = jobs.map(buildJobItemHTML).join('')
jobBody.set(jobTableRowsHTML)
```

File: jobs/index.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+<div id="tm-jobs_description" class="tm-jobs_description">
+  <button class="mdl-button mdl-js-button" onClick="JobBoard.collapse()">Back to jobs</button>
+  <hr>
+  <h2 id="tm-jobs_description_title"></h2>
+  <div id="tm-jobs_description_body"></div>
+</div>
+
+<script src="https://unpkg.com/showdown@1.7.6/dist/showdown.min.js"></script>
+<script src="https://unpkg.com/bluebird@3.5.1/js/browser/bluebird.min.js"></script>
```
Comment (theetrain) [inline jobs/index.html]: Use https://polyfill.io/v2/docs/api instead

**Comment by theetrain** [comment]
Made suggested changes. Sidenote regarding polyfill.io: after manually testing in IE11, I noticed the default minified file doesn't work, but specifying features does. Odd.

File: _includes/head.html [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <head>
   <meta charset="utf-8"/>
-  <meta http-equiv="X-UA-Compatible" content="IE=edge"//>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
```
Comment (hutchic) [inline _includes/head.html:3]: good catch 👍

**Review by hutchic — APPROVED** [review]

File: _includes/head.html (hunk 6 above)
Comment (theetrain) [reply _includes/head.html:3]: That wasn't me, that was https://github.com/prettier/prettier

Comment (hutchic) [reply _includes/head.html:3]: I like it when robots fix things :)

**Pull request merged**